src/curobo/wrap/reacher/motion_gen.py

```diff
@@ -1936,7 +1936,7 @@ def warmup(
                                     quaternion=state.ee_quat_seq)
                 start_state.position[...,
                                      warmup_joint_index] += warmup_joint_delta
-                for _ in range(3):
+                for _ in range(1):
```

Comment:
Do you have an estimate of how much this speeds up the warmup? Could you update the PR description with some details?

Also the rest of this commit diff looks like its just autoformat. If you can remove the diff I think that'd be better.

Reply:
Done